stdlib/multiprocessing/managers.pyi

```diff
@@ -66,6 +65,62 @@ class ValueProxy(BaseProxy, Generic[_T]):
     if sys.version_info >= (3, 9):
         def __class_getitem__(cls, item: Any) -> GenericAlias: ...
 
+class DictProxy(BaseProxy, MutableMapping[_KT, _VT]):
+    __builtins__: ClassVar[dict[str, Any]]
+    def __len__(self) -> int: ...
+    def __getitem__(self, __k: _KT) -> _VT: ...
+    def __setitem__(self, __k: _KT, __v: _VT) -> None: ...
+    def __delitem__(self, __v: _KT) -> None: ...
+    def __iter__(self) -> Iterator[_KT]: ...
+    def copy(self) -> dict[_KT, _VT]: ...
+    @overload
+    def get(self, __key: _KT) -> _VT | None: ...
+    @overload
+    def get(self, __key: _KT, __default: _VT | _T) -> _VT | _T: ...
+    @overload
+    def pop(self, __key: _KT) -> _VT: ...
+    @overload
+    def pop(self, __key: _KT, __default: _VT | _T) -> _VT | _T: ...
+    def keys(self) -> list[_KT]: ...  # type: ignore[override]
```

Comment:
This is because of https://github.com/python/cpython/blob/518595652759462b13904df767f69b8cc2c61143/Lib/multiprocessing/managers.py#L51 (took me a while to figure out)